src/app/Consts.js

```diff
@@ -0,0 +1,5 @@
+export const GameModes = {
+    PEOPLE: "People",
+    STARSHIPS: "Starships",
+    VEHICLES: "Vehicles"
+}
```

Comment:
The same const is on Filip's branch - FS_CreateFolders.

Reply:
I'll adjust my functions to use Justyna's consts during the next 'sprint' :)